### ISSUE TITLE
Change the application name - You are welcome to suggest a new name for Safe Eyes

### ISSUE THREAD

**Issue by slgobinath** [slgobinath/SafeEyes#99]
Hi all,
Recently I came across this article in [webupd8.org](http://www.webupd8.org/2017/03/terminix-terminal-emulator-renamed-to.html) regarding the name change from Terminix to Tilix due to some trademark issues. Also, I came across a security product `safeeyes`, when I searched `Safe Eyes` on Google, and feel it is better to change our name as early as possible.

What do you think?
I am waiting for your thoughts and name suggestions...
My suggestions:
 - EyeGuard

**Comment by JesseTG** [comment]
I don't see why not.  Maybe "EasyEye"?

**Comment by slgobinath** [comment]
Hi @orschiro @Yamakaky @hartwork and @radek-sprta,

I would like to hear your suggestions here :-)

Next version will include major changes like moving to Python 3 and changing the installation directory. So it is better to change the name as well.

**Comment by hartwork** [comment]
Hi!

How about just bumping the major version to 3 rather than a true rename? I would only recommend to rename if there are trademark issues or something winning the race on Google. Everyone who knew SafeEyes would have to learn the new name or may miss to get that update and you do have press coverage under that name already, so you do lose something. What's the gain?

Best, Sebastian

**Comment by orschiro** [comment]
- Eyeoe
- Eyelo
- EyeGo
- Eyesee (I see)

:-)

**Comment by xnakos** [comment]
First of all, great job on the application! Its definitely eye candy but there is also more to it than meets the eye. But what's with the "eye"? Isn't the application supposed to suggest non-targeted breaks, i.e. not only for the eyes, but also for posture and concentration? Anyway, I would suggest something in the lines of "Daybreaker". But then I may have misunderstood the purpose of the application, since I would also like the ability to select bigger break durations (much bigger than 120''), maybe 15' after 45' work, or 17' after 52' work (but then I would like my breaks to be organized by bigger breaks first (45'-15'), smaller breaks optional and second (10'' after 10' during the 45' of working time - a small break very close to the big break could be swallowed by the big break). Just food for thought. Thank you again! I will be spreading the word!

**Comment by hartwork** [comment]
@slgobinath, could you summarize the new situation? What's the new name?

**Comment by slgobinath** [comment]
Hi @hartwork,
I have postponed the idea until we get any legal issues because we have already created the repo in Ubuntu, Arch and Gentoo (thanks to you). Based on their site, that security product will be discontinued after May. So I guess there won't be any issues. If you have anything to add here, please go ahead.

Thanks

**Comment by hartwork** [comment]
Alright, thanks for the update.

**Comment by radek-sprta** [comment]
If any issues with the name arise in the future, I suggest EyeYoga as replacement :]
